src/main/resources/db/changelog/changelog_001.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.1.xsd">
+
+    <changeSet author="tolstiachok123" id="1">
+        <sql>
+            CREATE TABLE `author` (
+            `id` int NOT NULL,
+            `first_name` varchar(255) DEFAULT NULL,
+            `last_name` varchar(255) DEFAULT NULL,
+            PRIMARY KEY (`id`)
+            );
+        </sql>
+    </changeSet>
+
+    <changeSet author="tolstiachok123" id="2">
+        <sql>
+            CREATE TABLE `tag` (
+            `id` int NOT NULL,
+            `name` varchar(255) DEFAULT NULL,
+            PRIMARY KEY (`id`)
+            );
+        </sql>
+    </changeSet>
+
+    <changeSet author="tolstiachok123" id="3">
+        <sql>
+            CREATE TABLE `book` (
+            `id` int NOT NULL,
+            `created_at` datetime(6) DEFAULT NULL,
+            `image_url` varchar(255) DEFAULT NULL,
+            `price` decimal(19,2) DEFAULT NULL,
+            `title` varchar(255) DEFAULT NULL,
+            `updated_at` datetime(6) DEFAULT NULL,
+            `author_id` int DEFAULT NULL,
+            PRIMARY KEY (`id`),
+            KEY `FKklnrv3weler2ftkweewlky958` (`author_id`),
```

Comment:
Rename FK in more readable name
EXAMPLE:
KEY `book_ibfk_1` (`author_id`),
CONSTRAINT `book_ibfk_1` FOREIGN KEY (`author_id`) REFERENCES `author` (`id`)